src/Container/ApplicationFactory.php

```diff
@@ -136,6 +137,7 @@ public function __invoke(ContainerInterface $container)
 
         $app = new Application($router, $container, $finalHandler, $emitter);
 
+        $this->setPhpSettings($container);
```

Comment:
@Maks3w What do you think of moving `$this->setPhpSettings($container);` at the very begging of a `__invoke()` method, because it doesn't need `$app` at all?